docs/parser_translation.md

```diff
@@ -20,6 +20,9 @@ Prism::Translation::ParserCurrent.parse("puts 'Hello World!'")
 
 All the parsers are autoloaded, so you don't have to worry about requiring them yourself.
 
+If you also need to parse Ruby versions below 3.3 (which `prism` has no support for), check out
```

Comment:
I was surprised by this but indeed it looks like Prism has no support < 3.3: https://github.com/ruby/prism/pull/2364.